### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by akwei**
update readme
add architecture image
modify agent.properties item's comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,538 @@
...
+## QuickStart
+
+### Download
+Download `easeagent.jar` from releases [releases](https://github.com/megaease/easeagent/releases)
+
+or
 
 ```
-curl -Lk https://github.com/megaease/release/releases/download/easeagent/easeagent.jar -O
+curl -Lk https://github.com/megaease/easeagent/releases/download/{tag}/easeagent.jar -O
```
Comment (zhao-kun) [inline README.md:60]: I'd rather replace this with an inner architecture diagram of the EaseAgent.

File: README.md [hunk 2]
```diff
@@ -1,58 +1,538 @@
...
+- [Licenses](#Licenses)
 
-# Installation
 
+## Overview
 
-## Download 
+### Purpose & Principles
+EaseAgent is an APM tool under the Java system, used in a distributed system developed by Java.
+It provides cross-service call chain tracking and performance information collection for distributed systems, helps users analyze the overall structure of the distributed system and the call relationships between services and components, thereby improving the efficiency of troubleshooting.
```
Comment (zhao-kun) [inline README.md:32]: I roughly think the EaseAgent Principles are:
- No invasion
- Service-based view
-  High performance

File: README.md [hunk 3]
```diff
@@ -1,58 +1,538 @@
...
+  - job_name: 'user-app'
+    static_configs:
+    - targets: ['ip:easeagent.server.port']
+    metrics_path: "/prometheus/metrics"
+```
+Please note: Prometheus Label names must match the regex `[a-zA-Z_][a-zA-Z0-9_]*`. The character `-` will be replaced with `_`.
+
+See [Prometheus Metric names and labels rule](https://prometheus.io/docs/concepts/data_model)
+
+#### Metric Field
```
Comment (zhao-kun) [inline README.md:189]: The section of metric can be omitted in README.md, It should be written in the manual documents

File: README.md [hunk 4]
```diff
@@ -1,58 +1,538 @@
...
+    * `Http Servlet`、`Http Filter`
+    * `Spring Cloud Gateway`
+* Support `traceId` and `spanId` inject into user application
+* Support `health check` endpoint
+* Support `readiness check` endpoint for `SpringBoot2.2.x`
+
+## QuickStart
+
+### Download
+Download `easeagent.jar` from releases [releases](https://github.com/megaease/easeagent/releases)
```
Comment (zhao-kun) [inline README.md:55]: I'd rather replace this with an inner architecture diagram of the EaseAgent.

File: README.md [hunk 5]
```diff
@@ -1,58 +1,538 @@
...
 
+## Overview
 
-## Download 
+### Purpose & Principles
+EaseAgent is an APM tool under the Java system, used in a distributed system developed by Java.
+It provides cross-service call chain tracking and performance information collection for distributed systems, helps users analyze the overall structure of the distributed system and the call relationships between services and components, thereby improving the efficiency of troubleshooting.
+
+### Architecture Diagram
+![image](./doc/images/EaseAgent-Architecture.jpg)
```
Comment (zhao-kun) [inline README.md:35]: I'd rather replace this with an inner architecture diagram of the EaseAgent.

**Comment by akwei** [comment]
I add `development-guide.md` and `user-manual.md` to simplify `readme`

File: README.md [hunk 6]
```diff
@@ -1,58 +1,180 @@
...
+./prometheus --config.file=prometheus.yml
+```
+
+### Step 6
+Open Browser to visit [http://localhost:8080](http://localhost:8080)
 
-# Build From Source
+After visit more pages, open Prometheus manager [http://localhost:9090](http://localhost:9090), and search `_00GET__owners`.You will see as following
 
+![image](./doc/images/prometheus-demo-1.png)
```
Comment (zhao-kun) [inline README.md:149]: There is better than nothing, but I prefer to display metrics via the Grafana.

**Review by zhao-kun — APPROVED** [review]

**Pull request merged**